cleanup/artifactCleanup/artifactCleanup.groovy

```diff
@@ -176,11 +171,8 @@ if ( deprecatedConfigFile.exists() && configFile.exists() ) {
 private def artifactCleanup(String timeUnit, int timeInterval, String[] repos, log, paceTimeMS, dryRun = false, disablePropertiesSupport = false) {
     log.info "Starting artifact cleanup for repositories $repos, until $timeInterval ${timeUnit}s ago with pacing interval $paceTimeMS ms, dryrun: $dryRun, disablePropertiesSupport: $disablePropertiesSupport"
 
-    if ( !repos || (repos.length == 0) ) {
-        def errorMessage = 'repos parameter must be specified before initiating cleanup.'
-        log.error errorMessage
-        throw new CancelException(errorMessage, 400)
-    }
+    cancelOnNullrepos(repos,'repos parameter must be specified before initiating cleanup.' )
```

Comment:
Nitpick (not blocking from merge): inconsistent formatting for `cancelOnNullrepos`:

1. If camelCase, shouldn't function name be: `cancelOnNullRepos`?
2. Some calls have trailing space before `)` some don't. None has space between args.